components/Comments/CommentForm.vue

```diff
@@ -12,8 +12,8 @@
       <div class="comment-form-actions">
         <button type="button"
           class="button is-hidden-mobile"
-          :disabled="!this.hasContent"
-          @click="form.content = ''">
+          :disabled="this.isCommentFormOfContribution ? !this.hasContent : false"
```

Comment:
is probably equivalent to `this.isCommentFormOfContribution && !this.hasContent`